.github/workflows/ci.yaml

```diff
@@ -58,5 +58,18 @@ jobs:
     - name: Build
       run: make build
 
+    - name: Download MNist files
+      uses: suisei-cn/actions-download-file@v1
+      id: mnist 
+      with:
+          url: "https://github.com/srush/learns-dex/raw/main/mnist.bin"
+          target: examples/
+    - name: Download MNist labels file
+      uses: suisei-cn/actions-download-file@v1
+      id: labels 
+      with:
+          url: "https://github.com/srush/learns-dex/raw/main/labels.bin"
```

Comment:
Can't we use the official MNIST URLs?

Reply:
Also, you could add those two files to the list of files to be cached. This should skip the download in most cases.

Reply:
I'd also rather shell out to `wget` or `curl` instead of using an action template that doesn't look very well maintained 😬

Reply:
All three of these should be fixed now.